src/styles/watch.scss

```diff
@@ -0,0 +1,132 @@
+.watch {
+  box-sizing: border-box;
+  position: relative;
+  width: $watch-size;
+  height: $watch-size;
+  margin-inline: auto;
+  margin-top: calc((100vh / 2) - (#{$watch-size} / 2));
+
+  border: 5px solid $color-black;
+  border-radius: 50%;
+  background-color: $bg-color;
+  box-shadow: inset 0 0 48px $inner-shadow-color;
+
+  &__circle {
+    position: absolute;
+    left: 50%;
+    top: 50%;
+    transform: translate(-50%, -50%);
+
+    border-radius: 50%;
+
+    &--upper {
+      width: 7%;
+      height: 7%;
+
+      background-color: $color-black;
+      box-shadow: 0 0 10px $color-orange;
+    }
+
+    &--bottom {
+      width: 80%;
+      height: 80%;
+      text-align: center;
+
+      background-color: $bg-color-bottom;
+      box-shadow: 0 0 20px $color-orange;
+
+      &::after {
+        content: "x720";
+        position: absolute;
+        top: 75%;
+        left: 50%;
+        transform: translate(-50%, -50%);
+        font-size: 5vmin;
+
+        color: $color-orange;
+      }
+    }
+  }
+
+  &__marks {
+    position: absolute;
+    left: 50%;
+    width: 1%;
+    height: 100%;
+    z-index: 0;
+
+    background: $marks-color;
+
+    @each $num, $deg in $degs {
+      &--#{$num} {
+        transform: rotate(#{$deg}deg);
+      }
+    }
+  }
+
+  &__wrapper {
+    position: absolute;
+
+    &--marks {
+      width: 100%;
+      height: 100%;
+      z-index: 0;
+      transform: translateX(-0.5%);
+    }
+
+    &--hands {
+      width: 100%;
+      height: 100%;
+
+      &--sec {
+        transform: translateX(-0.5%);
+      }
+
+      &--min {
+        transform: translateX(-1%);
+      }
+
+      &--hours {
+        transform: translateX(-2%);
+      }
+    }
+  }
+
+  &__hand {
+    position: absolute;
+    left: 50%;
+    height: 50%;
+    transform-origin: 50% bottom;
+
+    animation-iteration-count: infinite;
+    animation-name: rotate;
+    animation-timing-function: steps(60);
+
+    &--sec {
+      width: 1%;
+      clip-path: polygon(50% 0, 50% 0, 100% 100%, 0 100%);
+
+      background-color: $color-sec;
+
+      animation-duration: #{60 / $speed-multiplier}s;
+    }
+
+    &--min {
+      width: 2%;
+      clip-path: polygon(50% 20%, 50% 20%, 100% 100%, 0 100%);
+
+      background-color: $color-min;
+
+      animation-duration: #{60 * 60 / $speed-multiplier}s;
```

Comment:
Try to avoid magic numbers. Create a variable for 60

Reply:
Done.